mwaa-local-env

```diff
@@ -76,6 +79,15 @@ reset-db)
 start)
    docker-compose -f ./docker/docker-compose-local.yml up
    ;;
+start-daemon)
```

Comment:
In the docs above you called this `start-silent` but here it is called `start-daemon`

Reply:
Works for me :)

Reply:
Please either update this line or the doc so that they are the same. Thanks!

Reply:
Done. Thanks!

Reply:
Let me know if there is anything else.